Add a few tests to verify basic usage. Additional contributions welcome.

	-mi

+++ tests/002.phpt	2015-04-13 20:40:26.000000000 -0400
@@ -0,0 +1,16 @@
+--TEST--
+Verify rejection of very simple password
+--SKIPIF--
+--POST--
+--GET--
+--FILE--
+<?php 
+if (crack_check("password")) {
+	echo "Bad, password 'password' accepted\n";
+} else {
+	echo "Good, password 'password' rejected\n";
+}
+
+?>
+--EXPECT--
+Good, password 'password' rejected
+++ tests/003.phpt	2015-04-13 20:48:28.000000000 -0400
@@ -0,0 +1,15 @@
+--TEST--
+Verify rejection of password identical to username
+--SKIPIF--
+--POST--
+--GET--
+--FILE--
+<?php 
+if (crack_check("b1934c823b137a492a13decfb939593e", "b1934c823b137a492a13decfb939593e", NULL)) {
+	echo "Bad, password identical to username accepted\n";
+} else {
+	echo "Good, password identical to username rejected\n";
+}
+?>
+--EXPECTREGEX--
+Good, password identical to username rejected
+++ tests/004.phpt	2015-04-13 20:40:43.000000000 -0400
@@ -0,0 +1,15 @@
+--TEST--
+Verify acceptance of good password
+--SKIPIF--
+--POST--
+--GET--
+--FILE--
+<?php 
+if (crack_check("6f763fbe906fc3c2fd57f3bcfa4afe79")) {
+	echo "Good, harsh password accepted\n";
+} else {
+	echo "Bad, password harsh rejected\n";
+}
+?>
+--EXPECT--
+Good, harsh password accepted
